include/deal.II/hp/refinement.h

```diff
@@ -293,28 +293,44 @@ namespace hp
      * cell diameter and $p_K$ the polynomial degree of the currently assigned
-     * finite element. Here, we assume that the finite element will not change
-     * in the adaptation process so that $p_K = \text{const}$. However during
-     * coarsening, the finite elements on siblings may be different, and their
-     * parent cell will be assigned to their least dominating finite element
-     * that belongs to its most general child. Thus, we will always interpolate
-     * on an enclosing finite element space. Additionaly assuming that the
-     * finite elements on the cells to be coarsened are sufficient to represent
-     * the solution correct (e.g. at least quadratic basis functions for a
-     * quadratic solution), we are confident to say that the error will not
-     * change by sole interpolation on the larger finite element space.
+     * finite element on cell $K$. Here, we assume that the finite element will
+     * not change in the adaptation process so that $p_K = \text{const}$.
+     * However during coarsening, the finite elements on siblings may be
+     * different, and their parent cell will be assigned to their least
+     * dominating finite element that belongs to its most general child. Thus,
+     * we will always interpolate on an enclosing finite element space.
+     * Additionaly assuming that the finite elements on the cells to be
+     * coarsened are sufficient to represent the solution correct (e.g. at least
+     * quadratic basis functions for a quadratic solution), we are confident to
+     * say that the error will not change by sole interpolation on the larger
+     * finite element space.
      *
-     * Further, we expect that the local error will be divided equally on
-     * all $2^{dim}$ children during refinement, whereas local errors on
-     * siblings will be summed up on the parent cell in case of coarsening. When
-     * transferring the predicted error to the coarsened mesh, make sure to
-     * configure your CellDataTransfer object with
-     * GridTools::CoarseningStrategies::sum() as a coarsening strategy.
+     * Further, the function assumes that the local error on a cell
+     * that will be refined, will lead to errors on the $2^{dim}$
+     * children that are all equal, whereas local errors on siblings
+     * will be summed up on the parent cell in case of
+     * coarsening. This assumption is often not satisfied in practice:
+     * For example, if a cell is at a corner singularity, then the one
+     * child cell that ends up closest to the singularity will inherit
+     * the majority of the remaining error -- but this function can
+     * not know where the singularity will be, and consequently
+     * assumes equal distribution.
+     *
+     * When transferring the predicted error to the coarsened mesh,
+     * make sure to configure your CellDataTransfer object with
+     * GridTools::CoarseningStrategies::sum() as a coarsening
```

Comment:
We moved the `CoarseningStrategies` namespace out of the `GridTools` one. I missed to adapt this part of the documentation at some point...
```suggestion
     * CoarseningStrategies::sum() as a coarsening
```